config/api.yml

```diff
@@ -117,6 +117,17 @@
     :klass: ChargebackRate
     :subcollections:
     - :rates
+  :container_deployments:
+    :description: Container Provider Deployment
+    :identifier: container_deployment
+    :klass: ContainerDeployment
+    :methods: *70174834084700
+    :options:
+    - :collection
```

Comment:
@alongoldboim I think the right thing to do here m ight be to put the configuration back to as cherry picked and try to fix the tests. The collections might need an extra line to authorize the test user, as we've done here: https://github.com/ManageIQ/manageiq/blob/darga/spec/requests/api/collections_spec.rb#L138